tests/ui/specs/SwitchItem/SwitchItem-specs.js

```diff
@@ -79,6 +79,7 @@ describe('SwitchItem', function () {
 			it('should re-select the item when selected twice', async function () {
 				await switchItem.focus();
 				await Page.spotlightSelect();
+				await switchItem.focus();
```

Comment:
I'm not sure that adding this step will solve the failed test. Failure did not occur after this issue was reported.
Is there a reason to refocus? Please let me know.

Reply:
I applied by mistake changes to the wrong test. I thought the one with 5-way fails and that it loses focus. I will check the pointer test and come up with a fix if it still fails. Thank you for noticing!